client/src/components/Header.js

```diff
@@ -98,7 +97,33 @@ const HeaderWrapper = styled.div`
   width: 100vw;
 `;
 
-export default ({ authLoading, onMenuClick, isAuthenticated }) => {
+export default ({ authLoading, onMenuClick, isAuthenticated, user }) => {
+  const menu = (
+    <Menu>
+      <Menu.Item>
+        <Link to="/profile">My Profile</Link>
+      </Menu.Item>
+      <Menu.Divider />
+      <Menu.Item>
+        <Link to="/create-organization-profile">Add Organisation</Link>
+      </Menu.Item>
+      <Menu.Divider />
+      {user
+        ? user.organizations.map((organization) => (
```

Comment:
instead of `{ variable ? <render stuff if value> : null }` you can do `{ variable && <render stuff />}`